### PR TITLE
Feature/awesome accesibility custom views

### PR DESCRIPTION
**Pull request by khaykov**
This PR fixes accessibility related keyboard lint issue, and custom view lint issues.

Custom view fixes include making gravatars clickable with TalkBack in notification elements.

` //noinspection AndroidLintClickableViewAccessibility` is used because for some reason SuppressLint is not working on this issue.

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/notifications/blocks/CommentUserNoteBlock.java [hunk 1]
```diff
@@ -76,12 +78,21 @@ public View configureView(View view) {
             String imageUrl = GravatarUtils.fixGravatarUrl(getNoteMediaItem().optString("url", ""), getAvatarSize());
             noteBlockHolder.avatarImageView.setImageUrl(imageUrl, WPNetworkImageView.ImageType.AVATAR);
             if (!TextUtils.isEmpty(getUserUrl())) {
+                noteBlockHolder.avatarImageView.setOnClickListener(new View.OnClickListener() {
```
Comment (malinajirka) [inline WordPress/src/main/java/org/wordpress/android/ui/notifications/blocks/CommentUserNoteBlock.java]: Shouldn't we set the onClickListener to null in both `else` branches, as the view can be recycled? (same applies to HeaderNoteBlock.java and UserNoteBlock.java)

File: WordPress/src/main/java/org/wordpress/android/ui/notifications/blocks/UserNoteBlock.java [hunk 2]
```diff
@@ -87,16 +89,25 @@ public View configureView(View view) {
             String imageUrl = GravatarUtils.fixGravatarUrl(getNoteMediaItem().optString("url", ""), getAvatarSize());
             noteBlockHolder.avatarImageView.setImageUrl(imageUrl, WPNetworkImageView.ImageType.AVATAR);
             if (!TextUtils.isEmpty(getUserUrl())) {
+                noteBlockHolder.avatarImageView.setOnClickListener(new View.OnClickListener() {
```
Comment (malinajirka) [inline WordPress/src/main/java/org/wordpress/android/ui/notifications/blocks/UserNoteBlock.java]: I just want to confirm, that we really need to set the onClickListener to the avatarImageView as we set an onClickListener to the root view few lines below. If we need it, we could probably reuse the `mOnClickListener` field.

Comment (khaykov) [reply WordPress/src/main/java/org/wordpress/android/ui/notifications/blocks/UserNoteBlock.java]: Yes, you are totally right. I just focused on gravatar, without realizing that tap on root view leads to the same place.

**Comment by khaykov** [comment]
no_idea

Thanks for checking! I probably should look at the code around the stuff I'm changing  a bit more :)

**Comment by malinajirka** [comment]
Perfect, thanks! It looks good.

**Pull request merged**